### DISCUSSION TITLE
2.5.299 Released 4 Years ago

### DISCUSSION THREAD

**Discussion by DarkDau** [requarks/wiki#6532] [General]
I discovered a funny little mistake and wanted to share it with you. No, it's not a translation error, the English locale shows exactly the same lapsus.
![wikijs_4_Jahre](https://github.com/requarks/wiki/assets/30416260/09d784ed-29c9-4575-ae8e-d8e91f041dbe)

**Comment by P4mes4n** [comment]
This is because your instance probably has no direct internet connection. I have the same behavior, but it works fine when you have direct internet connection.

**Reply by DarkDau** [reply]
I see.
